static/index.html

```diff
@@ -1,357 +1,7 @@
 <!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta name="description" content="Personal website for Aymeric Wibo"><meta name="viewport" content="width=device-width,initial-scale=1"><meta name="robots" content="index,follow"><meta name="theme-color" content="#000000"><link rel="icon" type="image/png" href="/public/icons/me.png"><link rel="manifest" href="manifest.json"><meta name="apple-mobile-web-app-capable" content="yes"><meta name="apple-mobile-web-app-status-bar-style" content="black-translucent"><meta name="apple-mobile-web-app-title" content="Aymeric Wibo"><title>Aymeric Wibo</title><style>@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Noto+Color+Emoji&display=swap');
 
```

Comment:
hmmmmmmmmmm there seems to be potential to save a couple more bytes here. interesting